tests/trainer/flags/test_overfit_batches.py

```diff
@@ -16,41 +16,47 @@
 from torch.utils.data.sampler import Sampler, SequentialSampler
 
 from pytorch_lightning import Trainer
+from tests.base.model_template import EvalModelTemplate
 from tests.helpers.boring_model import BoringModel, RandomDataset
 
 
-def test_overfit_multiple_val_loaders(tmpdir):
-    """Tests that overfit batches works with multiple val dataloaders."""
-    val_dl_count = 2
-    overfit_batches = 3
+@pytest.mark.parametrize("overfit_batches", [1, 2, 0.1, 0.25, 1.0])
+def test_disable_validation_when_overfit_batches_larger_than_zero(tmpdir, overfit_batches):
```

Comment:
I think we can remove this test completely. Testing within `tests/trainer/test_trainer_tricks.py:test_overfit_batch_limits` would be enough.

Reply:
Done